src/editor.cpp

```diff
@@ -375,13 +375,22 @@ void Editor::deleteWildMonGroup() {
 
     if (msgBox.clickedButton() == deleteButton) {
         auto it = project->wildMonData.find(map->constantName);
-        it.value().erase(labelCombo->currentText());
+        if (it == project->wildMonData.end()) {
+          logError(QString("Failed to find data for map %s. Unable to delete").arg(map->constantName));
```

Comment:
I don't think that `%s` is a valid format argument and would cause a crash.  To my knowledge, you can only use the numeric args, like `%1` or `%2`.  https://doc.qt.io/qt-5/qstring.html#arg

Same thing for the other QString below.